common/pom.xml

```diff
@@ -91,6 +103,14 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-redis</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-starter-config</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-starter-bus-amqp</artifactId>
+        </dependency>
```

Comment:
This looks good to me, but before we merge it into 'develop' have we tried all the services against a SNAPSHOT TDS_Common dependency version and made sure it stands up properly in docker-compose?